src/easylink/configuration.py

```diff
@@ -184,8 +184,12 @@ def _get_schema(self, potential_schemas: list[PipelineSchema]) -> PipelineSchema
         Returns
         -------
             The first ``PipelineSchema`` that validates the requested pipeline.
-            If no validated ``PipelineSchema`` is found, `exit()` is called with
-            `errno.EINVAL` and any validation errors are logged.
+
+        Raises
+        ------
+        SystemExit
+            If no requested pipeline is validated, all validation errors found are
+            batch-logged into a dictionary and the program exits with a non-zero code.
```

Comment:
batch-logged into a dictionary? What does this mean?

Reply:
I'm trying to explain that we're not simply raising on the first thing that doesn't validate but attempting to batch all errors together in to a single dictionary to return. I guess it's kinda redundat since I'm already saying "_all_ erros found..."?

Reply:
Changed